tutorials/step_by_step/simple_2d_game.rst

```diff
@@ -55,9 +55,10 @@ is defined, then input methods that trigger it are added.
 Open the project properties dialog again, but this time move to the
 "Input Map" tab.
 
+.. role:: python(code)
```

Comment:
The changes look good, but why define a role here? Note that all code snippets are defined as python code by default unless specified otherwise.